config/crd/bases/galaxy_v1beta1_galaxy_crd.yaml

```diff
@@ -230,9 +230,9 @@ spec:
                 description: Secret where the ingress TLS secret can be found
                 type: string
               nginx_client_max_body_size:
-                description: NGINX client max body size
-                default: 10m
-                type: string
+                description: Sets the maximum allowed size of the client request body in megabytes (defaults to 10M)
+                default: 10
+                type: integer
```

Comment:
The galaxy-operator change would be a breaking change for anyone who is currently setting `nginx_client_max_body_size` on their Galaxy.spec right if I understand correctly.

I don't think we should do a breaking change like this until we can cut a new apiVersion for the Galaxy CRD.  So I think we need to make it backwards compatible if we are going to merge this.  And since the validation happens on the CRD when it is applied, I don't see a clean way to add backwards compatibility without adding a new apiVersion.